scopes/react/react/react.ui.runtime.tsx

```diff
@@ -32,12 +32,10 @@ export class ReactUI {
       );
     });
 
-    if (typeof window !== 'undefined' && window.location.search.includes('highlighter')) {
-      compositionsUI.registerMenuWidget({
-        location: 'start',
-        content: <HighlighterWidget />,
-      });
-    }
+    compositionsUI.registerMenuWidget({
```

Comment:
are we sure it runs on ssr? please make sure before removing the check for `window`